app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
Ways to implement views in alembic: http://alembic.zzzcomputing.com/en/latest/cookbook.html